### ISSUE TITLE
Loinc 12962-7:  "Urea nitrogen [Mass/volume] in Venous blood"

### ISSUE THREAD

**Issue by kingmanzhang** [obophenotype/human-phenotype-ontology#4134]
Suggest creating a new child term of HP:0031970 [Abnormal blood urea nitrogen] for Loinc 12962-7 [Urea nitrogen [Mass/volume] in Venous blood]
New term label: Abnormal venous blood urea nitrogen, then its children (increased, decreased)
New term comment (if any):
Your biocurator ID for loinc2hpo (if desired): JGM:azhang

**Comment by pnrobinson** [comment]
@kingmanzhang 
this is https://hpo.jax.org/app/browse/term/HP:0031970
it is normally determined in venous blood and so we do not need to specify this in the term

**Comment by kingmanzhang** [comment]
Done.
